security/compliance_operator/compliance-operator-installation.adoc

```diff
@@ -0,0 +1,73 @@
+[id="compliance-operator-installation"]
+= Installing the Compliance Operator
+include::modules/common-attributes.adoc[]
+:context: compliance-advanced
+
+toc::[]
+
+
+Before you can actually use the operator, you need to make sure it is deployed in the cluster. Depending on your needs, you might want to deploy the upstream released packages or directly from the source.
```

Comment:
I would say that we only instruct folks on how to use the released packages. As it gets easier for us to support.